embedded-graphics/src/primitives/triangle/styled.rs

```diff
@@ -216,4 +216,35 @@ mod tests {
             .into_iter()
             .map(|Pixel(p, col)| Pixel(p - Point::new(0, 35), col))));
     }
+
+    #[test]
+    fn styled_stroke_equals_lines() {
+        let triangle = Triangle::new(Point::new(10, 10), Point::new(20, 20), Point::new(30, -30));
+
+        let styled = triangle.into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 1));
+
+        let mut tri_display: MockDisplay<BinaryColor> = MockDisplay::new();
+        tri_display.set_allow_out_of_bounds_drawing(true);
```

Comment:
Wouldn't it be better to choose a triangle that is inside the `MockDisplay` bounds and keep the bounds checking enabled? Otherwise parts of the triangle wont be checked.